src/NHibernate.Test/TestDialects/MsSql2005TestDialect.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace NHibernate.Test.TestDialects
+{
+	public class MsSql2005TestDialect : TestDialect
+	{
+		public MsSql2005TestDialect(Dialect.Dialect dialect)
+			: base(dialect)
+		{
+		}
+
+		public override bool SupportsAggregateInSubSelect => false;
```

Comment:
It seems that majority of the dialects do not support this behaviour. Wouldn't it be more sensible to change the default to `false`?

Reply:
I agree, changed default to false.